tutorials/parser/src/main/resources/META-INF/csvParserProperties.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
```

Comment:
Will this properties file actually take effect when run on a cluster ? Shouldn't the file be named
properties-csvParserProperties.xml ?

Please test all examples both in local mode as well as on a cluster -- we want the user experience to be smooth and painless when running these.